RELEASE.md

```diff
@@ -0,0 +1,256 @@
+
+1. CHANGELOG: go through the CHANGELOG.md file found in the root folder and verify all required changes / bug fixes / features for this release are found in the file. Add a headline of the version and the release date to close all of its changes, for example for releasing version 0.3.0:
+
+    ### 0.3.0 (2024-03-25)
+    #### Changes
+    * Python: Allow routing Cluster requests by address. ([#1021](https://github.com/aws/glide-for-redis/pull/1021))
+    * Python, Node: Added HSETNX command. ([#954](https://github.com/aws/glide-for-redis/pull/954), [#1091](https://github.com/aws/glide-for-redis/pull/1091))
+
+    #### Features
+    * Python: Allow chaining function calls on transaction. ([#987](https://github.com/aws/glide-for-redis/pull/987))
+
+    ### 0.2.0 (2024-02-11)
+    #### Changes
+    * Python, Node: Added ZCARD command ([#871](https://github.com/aws/glide-for-redis/pull/871), [#885](https://github.com/aws/glide-for-redis/pull/885))
+
+    b. Open a PR to the main branch with the missing changes and the new version header, and continue after it’s merged.
+
+
+2. **If it’s a new minor/major version**, create a new branch named in the format `v$MAJOR_VERSION.$MINOR_VERSION` (e.g., v0.2), and checkout only the relevant code for this release (probably the latest commit in main, including the CHANGELOG latest commit). <br>
+
+    MAJOR_VERSION = 0 <br>
+    MINOR_VERSION = 3 <br>
+    git checkout -b `v$MAJOR_VERSION.$MINOR_VERSION` <br>
+
+    a. Find how you saved the upstream remote repo (valkey-io/valkey-glide):
+    ```
+    [10:09:23] ubuntu (v1.x*)$ git remote -vv
+    origin  https://github.com/your-repo/valkey-glide (fetch)
+    origin  https://github.com/your-repo/valkey-glide (push)
+    origin-aws      https://github.com/valkey-io/valkey-glide (fetch)
+    origin-aws      https://github.com/valkey-io/valkey-glide (push)
+   ```
+
+   b. Push this branch to the upstream repo:
+   git push origin-aws
+
+    1. **If it’s only a patch version and the branch already exists**, checkout the existing branch (e.g. git checkout origin-aws/v0.2), and merge in the commit you’ve merge into main with the updates to the CHANGELOG.md file.
+    2. **Attributions file (ORT)**: Make sure the attributions file is updated by manually running the ORT workflow:
+
+        a. Go to: https://github.com/valkey-io/valkey-glide/actions/workflows/ort.yml <br>
+
+        b. On the right corner, click on “Run workflow” and set the branch to run against the ORT tool to your newly created branch name (e.g. v0.3), and the exact release version (e.g. 0.3.0) <br>
+
+        c. <br>
+        ![OSS Review Toolkit](images-release-md/oss-review-toolkit.png)
+
+        d. If diff is found the workflow will open a PR in the release branch with the required changes. Make sure that all dependencies are approved by a Valkey-GLIDE maintainer, refer to the SOP - Check the Licenses: ORT/Attributions File Setup and Usage Instructions. Merge the PR once it is ready.
```

Comment:
SOP is one of the quip files mentioned in the original documents. i referenced it here in case they get contacted and will need to refer to this specific internal document to resolve issue

Reply:
we cannon refer non-open-source files here